### PR TITLE
Implementation ABAQUS inp format input/output

### PR DESCRIPTION
**Pull request by hillyuan**
Enable ABAQUS inp files' input and output. 
*  Only mesh information (*Node and *Element) are considered
*  Pytest passed

### CODE REVIEW

**Comment by nschloe** [comment]
Great contribution!

Since recently, meshio uses [black](https://github.com/ambv/black), which is what the tests are complaining about at the moment. Simply run
```
black meshio/ test/*.py
```
to have your files reformatted accordingly. If you want you can try
```
make lint
```
to see that it worked.

File: README.md [hunk 1]
```diff
@@ -1,108 +1,110 @@
...
-2. tag and upload to PyPi:
-    ```
-    make publish
-    ```
-
-### License
-
-meshio is published under the [MIT license](https://en.wikipedia.org/wiki/MIT_License).
+# meshio
+
```
Comment (nschloe) [inline README.md]: A lot of whitespace changes happened in the README, and I'm not sure what's up. Windows line breaks perhaps?

File: README.md [hunk 2]
```diff
@@ -1,108 +1,110 @@
...
+ * [Medit](https://people.sc.fsu.edu/~jburkardt/data/medit/medit.html)
+ * [MED/Salome](http://docs.salome-platform.org/latest/dev/MEDCoupling/med-file.html)
+ * [Gmsh](http://gmsh.info/doc/texinfo/gmsh.html#File-formats)
+ * [OFF](http://segeval.cs.princeton.edu/public/off_format.html)
+ * [PERMAS](http://www.intes.de)
+ * [STL](https://en.wikipedia.org/wiki/STL_(file_format))
+ * [VTK](https://www.vtk.org/wp-content/uploads/2015/04/file-formats.pdf)
+ * [VTU](https://www.vtk.org/Wiki/VTK_XML_Formats)
+ * [XDMF](http://www.xdmf.org/index.php/XDMF_Model_and_Format)
+ * [ABAQUS inp](http://abaqus.software.polimi.it/v6.14/index.html)
```
Comment (nschloe) [inline README.md]: Best keep this list in alphabetic order.

Also, how about calling this format just `Abaqus` (no capitalization, no `inp`)?

File: tools/meshio-convert [hunk 3]
```diff
@@ -87,7 +87,7 @@ def _parse_options():
         "--input-format",
         "-i",
         type=str,
-        choices=meshio.input_filetypes,
+        choices=meshio.helpers.input_filetypes,
```
Comment (nschloe) [inline tools/meshio-convert]: Well spotted!

**Comment by hillyuan** [comment]
I am sorry I cannot find where README.md wrong. Thanks for your modification

**Comment by nschloe** [comment]
As I had thought, the line breaks were converted to Windows line breaks. Fixed that now.

**Pull request merged**